src/libraries/System.ComponentModel.Composition/ref/System.ComponentModel.Composition.csproj

```diff
@@ -2,15 +2,9 @@
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppMinimum);netstandard2.0</TargetFrameworks>
   </PropertyGroup>
+
   <ItemGroup>
     <Compile Include="System.ComponentModel.Composition.cs" />
     <Compile Include="System.ComponentModel.Composition.Forwards.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">
-    <ProjectReference Include="$(LibrariesProjectRoot)System.Linq.Expressions\ref\System.Linq.Expressions.csproj" />
```

Comment:
If we are removing P2Ps from netcoreappcurrent build, how are we supposed to consume the live build of S.L.Expressions?

Reply:
The targeting pack that is resolved is the live built one for NetCoreAppCurrent which resides under `artifacts\bin\microsoft.netcore.app.ref\`

Reply:
This is already the current behavior in main. Let me explain it in more details:
1. The shared framework libraries (both ref and src) get built as part of the libs.sfx subset.
2. These projects binplace their built assets into the live targeting pack and runtime pack folders.
3. NetCoreAppCurrent out-of-band libraries build on top of the targeting pack and/or runtime pack and receive that pack's references.

ProjectReference items are used to reference live built assets that aren't part of the targeting pack (OOBs) or when the library is available in the targeting pack but ships via a package as well, i.e. System.Text.Json. In the case of such special libraries like STJ, we reference them via ProjectReferences so that the library has a package dependency towards STJ.